PacketTunnelProvider/NetworkProtection/VPNPrivacyConfigurationManager.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  VPNPrivacyConfigurationManager.swift
+//  DuckDuckGo
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import Core
+import BrowserServicesKit
+import Combine
+import Common
+
+public final class VPNPrivacyConfigurationManager: PrivacyConfigurationManaging {
+
+    static let shared = VPNPrivacyConfigurationManager()
+
+    private let lock = NSLock()
+
+    var embeddedConfigData: Data {
+        let configString = """
+    {
+            "readme": "https://github.com/duckduckgo/privacy-configuration",
+            "version": 1693838894358,
+            "features": {
+                "networkProtection": {
+                    "state": "enabled",
+                    "exceptions": [],
+                    "settings": {}
+                }
+            },
+            "unprotectedTemporary": []
+        }
+    """
+        let data = configString.data(using: .utf8)
+        return data!
+    }
```

Comment:
I think we should close this gap as quickly as possible; from my perspective we can live with this in the short term, at least as part of running the pixel rollout test, but I'd advocate for us getting real embedded data working ASAP. Would you be able to scope the changes required for this for both iOS and macOS?

Reply:
Follow up opened here: https://app.asana.com/0/1200890834746050/1208305934034181/f